### PR TITLE
Change "application" to "integration".

### PR DESCRIPTION
**Pull request by RukshiW**

### CODE REVIEW

**Comment by RukshiW** [comment]
Closes https://github.com/wso2-enterprise/choreo/issues/3898

File: en/docs/integrations/integration-concepts.md [hunk 1]
```diff
@@ -8,18 +8,18 @@ Integration in Choreo is the process of connecting APIs, events, and streams to
...
- - **Salesforce**: To trigger an application based on topics in a Salesforce account. 
+ - **Schedule**: To trigger an integration based on a schedule that you set.
+ - **Manual**: To run and test an integration manually.
+ - **Calendar**: To trigger an integration based on events in a Google Calendar.
+ - **GitHub**: To trigger an integration based on GitHub events and actions.
+ - **Salesforce**: To trigger an integration based on topics in a Salesforce account. 
 
 ## API Call
-An API call in Choreo allows you to connect to and interact with an external service through your application. You can use an API call to interact with generic protocol-based APIs and application-specific APIs.
+An API call in Choreo allows you to connect to and interact with an external service through your integration. You can use an API call to interact with generic protocol-based APIs and application-specific APIs.
```
Comment (shaniR) [inline en/docs/integrations/integration-concepts.md:22]: we need to make this generic right? cz we have API call in services as well..

File: en/docs/integrations/integration-concepts.md [hunk 2]
```diff
@@ -40,16 +40,16 @@ Following are the connections you can use to interact with application-specific
  - **NetSuite**: To interact with the NetSuite API and perform required operations. 
 
 ## Statement
-A statement is a syntactic unit that specifies the action to be carried out in a Choreo application. 
+A statement is a syntactic unit that specifies the action to be carried out in an integration.
```
Comment (shaniR) [inline en/docs/integrations/integration-concepts.md:43]: same as above

File: en/docs/integrations/integration-concepts.md [hunk 3]
```diff
@@ -40,16 +40,16 @@ Following are the connections you can use to interact with application-specific
  - **NetSuite**: To interact with the NetSuite API and perform required operations. 
 
 ## Statement
-A statement is a syntactic unit that specifies the action to be carried out in a Choreo application. 
+A statement is a syntactic unit that specifies the action to be carried out in an integration. 
 
 Choreo supports the following statements:
 
- - **Log**: To add an entry to the application log at runtime. You can either add an **Info** log or an **Error** log depending on your requirement.   
+ - **Log**: To add an entry to the integration log at runtime. You can either add an **Info** log or an **Error** log depending on your requirement.
```
Comment (shaniR) [inline en/docs/integrations/integration-concepts.md:47]: do we call it the integration log?

File: en/docs/services/create-your-first-service.md [hunk 4]
```diff
@@ -103,11 +103,11 @@ To create the service, follow the procedure below:
     
 Now you have completed designing your service. It looks as follows.
 
-- In the Low Code View
+- In the low-code View
```
Comment (shaniR) [inline en/docs/services/create-your-first-service.md:106]: isn't the low-code view = code view + graphical one?

**Review by shaniR — CHANGES_REQUESTED** [review]

File: en/docs/integrations/integration-concepts.md (hunk 3 above)
Comment (RukshiW) [reply en/docs/integrations/integration-concepts.md]: Changed it to just "log".

File: en/docs/integrations/integration-concepts.md (hunk 1 above)
Comment (RukshiW) [reply en/docs/integrations/integration-concepts.md]: Changed.

**Review by shaniR — APPROVED** [review]

**Pull request merged**